chain/lf3/participation.go

```diff
@@ -82,8 +83,12 @@ func (p *Participant) Stop(ctx context.Context) error {
 
 func (p *Participant) run(ctx context.Context) (_err error) {
 	defer func() {
-		if ctx.Err() == nil && _err != nil {
-			log.Errorw("F3 participation stopped unexpectedly", "error", _err)
+		if ctx.Err() != nil {
+			_err = nil
```

Comment:
We could also check if the returned error is "context canceled" _and_ the passed context has been canceled?